apps/hobbies-helsinki/.prettierrc.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Prettier has different rules than in root folder

Reply:
Removed from hobbies